src/ReactiveUI/Routing/RoutingState.cs

```diff
@@ -83,14 +85,14 @@ public IScheduler Scheduler
         /// Gets or sets a command which will navigate back to the previous element in the stack.
         /// </summary>
         [IgnoreDataMember]
-        public ReactiveCommand<Unit, Unit> NavigateBack { get; protected set; }
+        public ReactiveCommand<Unit, Unit>? NavigateBack { get; protected set; }
```

Comment:
Don't think these ones should ever be null. It's due to the `SetupRx()` method being called in the constructor. Let's just make sure they are constructed.